src/index.html

```diff
@@ -28,14 +28,16 @@ <h1>Today's To Do</h1>
             <img src="./assets/return.svg" alt="return icon" class="return" />
           </div>
         </form>
-        <hr />
-        <ul class="todo__list">
-          <!-- Dynamicaly Generated -->
-        </ul>
+        <!-- <hr /> -->
```

Comment:
- [ ] Kindly remove all unused code this will make your code mush noise, It is always a good practice to keep your code clean from such comments.